### PR TITLE
The great src. purge

### PR DESCRIPTION
**Pull request by ghost**
Removed all non-cache ```src.```s from the code

No runtimes currently, except unrelated ones

### CODE REVIEW

**Comment by KorPhaeron** [comment]
We're trying to get two big PRs over the finish line right now including one by a first time contributor it'd be nice if they didn't need to fix 3k lines worth of potential conflicts on top of that

Would prefer this gets merged after circuits and techwebs do

File: code/controllers/admin.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@ INITIALIZE_IMMEDIATE(/obj/effect/statclick)
 /obj/effect/statclick/Initialize(mapload, text, target) //Don't port this to Initialize it's too critical
 	. = ..()
 	name = text
-	src.target = target
+	target = target
```
Comment (Cyberboss) [inline code/controllers/admin.dm]: Can already tell this PR is garbage

**Comment by Cyberboss** [comment]
This sort of thing hasn't been done already because every single src. replacement needs auditing to make sure it's not a case that I pointed out above. As it stands, this is just guaranteed to introduce impossible to find bugs with no real positive

**Comment by MoreRobustThanYou** [comment]
@Cyberboss I DID manage to cover the most common src caches.

**Pull request closed without merging**